source/views/sis/course-search/index.js

```diff
@@ -0,0 +1,226 @@
+// @flow
```

Comment:
Please rename this file to something else… `search`, maybe.

`index` should only hold the exports that make up the public interface to the folder.

(it's an ideal I'm slowly working towards)